### ISSUE TITLE
bad path to vendor font

### ISSUE THREAD

**Issue by bryanlandia** [raccoongang/xblock-video#221]
In `static/vendor/css/video-js.min.css`

```
@font-face{font-family:VideoJS;src:url(font/VideoJS.eot?#iefix) 
```

`font/` should be `fonts/`

**Comment by bryanlandia** [comment]
or maybe the other way around, changing the folder name

**Comment by z4y4ts** [comment]
Hi,

Actually, from what I can see this font file is not being used. We style player with font-awesome to mimic native edX player.

So it's a bit hard to reproduce this issue. But looking at `video-js.min.css` now, I can see it's better to move font to `font/` subfolder. Also, other font files can be removed.


Do you have anything broken by font file misplacement?


Best regards,
Alexander

**Comment by z4y4ts** [comment]
The fix is now part of newly released [v0.7.0](https://github.com/raccoongang/xblock-video/releases/tag/v0.7.0)
